src/localconf.js

```diff
@@ -269,6 +269,7 @@ function buildConfig(opts) {
     workspaceLaunchUrl,
     workspaceTerminateUrl,
     homepageChartNodes: components.index.homepageChartNodes,
+    customHomepageChartConfig: components.index.customHomepageChartConfig,
```

Comment:
Do we have some documentation for the config change? (I actually don't know where those docs would be)